style.css

```diff
@@ -320,3 +338,252 @@ nav {
   margin-top: 50px;
   margin-bottom: 10px;
 }
+
+.snap01 {
+  display: none;
+}
+
+.list-1 {
+  list-style: none;
+}
+
+.list-item-1 {
+  display: flex;
+  justify-content: space-between;
+}
+
+.list-2 {
+  display: flex;
+  flex-direction: column;
+  flex-basis: 100px;
+  list-style: none;
+}
+
+.list-item-2 {
+  background: #f7f7f9;
+  display: flex;
+  align-items: center;
+  padding: 10px;
+  margin-top: 20px;
+  border-radius: 5px;
+}
+
+.name {
+  margin-left: 10px;
+}
+
+.list-3 {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 30px;
+}
+
+.list-4 {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+@media screen and (min-width: 1024px) {
+  .logo {
+    margin-left: 60px;
+    margin-top: 10px;
+  }
+
+  .mobile-nav {
+    display: none;
+  }
+
+  .desktop-nav {
+    display: inherit;
+    margin-right: 30px;
+  }
+
+  .nav-link {
+    text-decoration: none;
+    margin-right: 30px;
+    color: #344563;
+  }
+
+  .homepage {
+    height: 768px;
+    background-image: url(./images/background_1024.png);
+  }
+
+  .home-page {
+    width: 870px;
+    margin: 100px auto;
+  }
+
+  .intro {
+    width: 350px;
+    height: 122px;
+  }
+
+  /* .description {
+    width: 870px;
+    height: 60px;
+  } */
+
+  .cr-grid {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .w1contain {
+    flex-direction: row;
+    justify-content: space-between;
+    align-self: center;
+  }
+
+  .for-fullscreen {
+    margin-left: 50px;
+  }
+
+  .for-portfolio {
+    margin: 20px 50px;
+  }
+
+  .pti {
+    display: none;
+  }
+
+  .snap01 {
+    display: inherit;
+  }
+
+  .img13 {
+    align-self: flex-start;
+  }
+
+  .img24 {
+    align-self: flex-end;
+  }
+
+  .w1par {
+    width: 496px;
+  }
+
+  .abm-intro {
+    width: 200px;
+    height: 114px;
+  }
+
+  .w1btn3 {
+    width: 162px;
+    height: 48px;
+  }
+
+  .section-about-myself {
+    width: 40%;
+    margin-left: 70px;
+  }
+
+  .section-about-myself-2 {
+    width: 60%;
+    margin-left: 20px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .list-item-1 {
+    margin-left: 20px;
+  }
+
+  .list-3 {
+    margin-left: 20px;
+  }
+
+  .list-2 {
+    flex-direction: row;
+    margin: 25px 10px 30px 10px;
+  }
+
+  .list-item-2 {
+    width: 100px;
+    margin: 10px;
+    display: list-item;
+    margin-top: -20px;
+  }
+
+  .name {
+    margin-left: 10px;
+  }
+
+  .abm-container {
+    flex-direction: row;
+    justify-content: space-around;
+  }
```

Comment:
- [ ] Consider making your `.abm-container`  and `.contact-info` design to match that of your chosen template for the desktop and mobile version by adding the correct `border-radius`.

See Figma design:

<img width="72" alt="chrome_AF2jYVBPFY" src="https://user-images.githubusercontent.com/81468675/144474602-3de55eee-cb30-44e0-8363-b0e2b41fb3be.png">

<img width="65" alt="chrome_TOMysjob40" src="https://user-images.githubusercontent.com/81468675/144475062-38be52ef-2aee-4c60-9ff2-fbf6b3d56285.png">